OurUmbraco.Site/Views/Master.cshtml

```diff
@@ -71,11 +71,24 @@
 
             if (string.IsNullOrWhiteSpace(title) == false)
             {
-                <text>@title - </text>
+                if (Model.Content.Name.ToString().ToLower() == "documentation")
+                {
+                    var path = title.Split(new string[] { " - " }, StringSplitOptions.None);
+
+                    System.Globalization.TextInfo myTI = new System.Globalization.CultureInfo("en-US", false).TextInfo;
+
+                    var firstUrlPath = myTI.ToTitleCase(path[0]);
+                    var lastUrlPath = myTI.ToTitleCase(path[path.Length - 1]);
```

Comment:
Maybe also replace `-` with an empty string?

Reply:
Not sure what you mean? 🙂

Reply:
Now the title will be `System-Requirements, Getting-Started`, why not `System requirements, Getting started`?